Doc/library/platform.rst

```diff
@@ -253,3 +253,26 @@ Unix Platforms
    using :program:`gcc`.
 
    The file is read and scanned in chunks of *chunksize* bytes.
+
+
+Linux Platforms
+---------------
+
+.. function:: freedesktop_osrelease()
+
+   Get operating system identification from ``os-release`` file and return
+   it as a dict. The ``os-release`` file is a `freedesktop.org standard
+   <https://www.freedesktop.org/software/systemd/man/os-release.html>`_ and
+   supported by majority of Linux distributions. All fields except ``NAME``
+   and ``ID`` are optional. If present, the ``ID_LIKE`` parsed and presented
+   as a tuple of strings.
+
+   Note that fields like ``NAME``, ``VERSION``, and ``VARIANT`` are strings
+   suitable for presentation to users. Programs should use fields like
+   ``ID`` + ``ID_LIKE``, ``VERSION_ID``, or ``VARIANT_ID`` to identify
+   Linux distributions. Vendors may include additional fields.
+
```

Comment:
Would you mind to document that invalid lines are silently ignored? I understood that only 3 formats are valid:

```
NAME=value
NAME="value"
NAME='value'
```

NAME must be non-empty and only made of characters: a-z, A-Z, 0-9 or "_". I don't know if it's worth it to describe the format here.

Also, please mention the encoding used to decode the file content (UTF-8).